pkg/scaffold/olm-catalog/csv.go

```diff
@@ -70,8 +87,10 @@ func (s *CSV) GetInput() (input.Input, error) {
 // CustomRender allows a CSV to be written by marshalling
 // olmapiv1alpha1.ClusterServiceVersion instead of writing to a template.
 func (s *CSV) CustomRender() ([]byte, error) {
+	s.initFS(afero.NewOsFs())
```

Comment:
Is this the only place `s.initFS()` is called? I was expecting to see it called in a test with `afero.NewMemMapFs()` based on your comment in line 56. If that's coming in a follow-on PR, that's fine. Just want to make sure I'm not missing something.

Reply:
Yeah I now call `initFS` in `getFS()` below and in [CSV tests](https://github.com/operator-framework/operator-sdk/pull/1016/files#diff-710dfe5d2c7daf445ec619270f7cfc35R114).